### PR TITLE
Update gendependabot to add rule for dependabot to update docker dependencies

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -262,18 +262,26 @@ build-binary-internal:
...
+	@echo "  - package-ecosystem: \"${PACKAGE}\"" >> ${DEPENDABOT_PATH};
+	@echo "    directory: \"${DIR}\"" >> ${DEPENDABOT_PATH};
+	@echo "    schedule:" >> ${DEPENDABOT_PATH};
+	@echo "      interval: \"weekly\"" >> ${DEPENDABOT_PATH};
+
 .PHONY: gendependabot
 gendependabot:
-	@echo "Recreate dependabot.yml file"
-	@echo "# File generated by \"make gendependabot\"; DO NOT EDIT.\n" > ${DEPENDABOT_PATH}
+	@echo "Recreatings ${DEPENDABOT_PATH} file"
```
Comment (mx-psi) [inline Makefile:276]:
```suggestion
	@echo "Recreating ${DEPENDABOT_PATH} file"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,11 +1,18 @@
-# File generated by "make gendependabot"; DO NOT EDIT.
```
Comment (mx-psi) [inline .github/dependabot.yml:1]: Shouldn't this be kept?

**Comment by tigrannajaryan** [comment]
Please add a PR description to clarify what problem this solves.

File: .github/dependabot.yml (hunk 2 above)
Comment (bogdandrutu) [reply .github/dependabot.yml]: Done.

**Review by mx-psi — APPROVED** [review]

**Review by jrcamp — APPROVED** [review]

**Pull request merged**